dopamine_BL21/11/11_stats.jl

```diff
@@ -0,0 +1,140 @@
+# ────────────────────────────────────────────────────────────────────────────────────────
+
+# CB on and off bacteria vs N2 on and off bacteria
+# ie. speed diff of CB vs speed diff of N2
+# p = <1e-09 --> significant difference, as expected
+
+# MT on and off bacteria vs N2 on and off bacteria
+# ie. speed diff of MT vs speed diff of N2
+# p = <1e-06 --> significant difference, as expected
+
+
+
+# comparing across strains for dopamine medium
+lm(@formula(meanspeed ~ worm*bacteria), subset(trackstats, :medium=>ByRow(==("DA"))))
+# meanspeed ~ 1 + worm + bacteria + worm & bacteria
+
+# Coefficients:
+# ───────────────────────────────────────────────────────────────────────────────────────────
+#                                 Coef.  Std. Error       t  Pr(>|t|)   Lower 95%   Upper 95%
+# ───────────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                 236.311       8.82898   26.77    <1e-57   218.862    253.76
+# worm: CB                      7.86592    11.9762     0.66    0.5123   -15.8032    31.5351
+# worm: MT                    -25.0427     12.2116    -2.05    0.0421   -49.177     -0.908308
+# bacteria: BL21             -118.027      11.3625   -10.39    <1e-18  -140.483    -95.5704
+# worm: CB & bacteria: BL21    -3.19201    16.1265    -0.20    0.8434   -35.0635    28.6795
+# worm: MT & bacteria: BL21    33.5566     16.2246     2.07    0.0404     1.49108   65.6221
+# ───────────────────────────────────────────────────────────────────────────────────────────
+
+# CB on and off bacteria vs N2 on and off bacteria
+# ie. speed diff of CB vs speed diff of N2
+# p = 0.8434 --> not significant, as expected
+
+# MT on and off bacteria vs N2 on and off bacteria
+# ie. speed diff of MT vs speed diff of N2
+# p = 0.0404 --> BARELY SIGNIFICANT, KINDA EXPECTED?
+
+
+
+# comparing across mediums for N2 strain
+lm(@formula(meanspeed ~ medium*bacteria), subset(trackstats, :worm=>ByRow(==("N2"))))
+# meanspeed ~ 1 + medium + bacteria + medium & bacteria
+
+# Coefficients:
+# ──────────────────────────────────────────────────────────────────────────────────────────
+#                                  Coef.  Std. Error       t  Pr(>|t|)  Lower 95%  Upper 95%
+# ──────────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                   292.938      10.4741   27.97    <1e-45    272.132   313.743
+# medium: DA                    -56.6264     13.903    -4.07    <1e-04    -84.243   -29.0097
+# bacteria: BL21               -171.053      13.3124  -12.85    <1e-21   -197.497  -144.61
+# medium: DA & bacteria: BL21    53.0266     17.7668    2.98    0.0036     17.735    88.3182
+# ──────────────────────────────────────────────────────────────────────────────────────────
+
+# dopamine vs. buffer, bacteria vs. no bacteria
+# ie. speed diff of N2 w buffer vs. w dopamine
+# p = 0.0036 --> SIGNIFICANT, NOT AS EXPECTED
```

Comment:
N2
dopamine vs. buffer, bacteria vs. no bacteria
p = 0.0036 --> SIGNIFICANT, NOT AS EXPECTED

Reply:
Which? The dop v buffer, or the +/- bacteria?

Reply:
speed diff of N2 +/- bacteria on dopamine vs. on buffer
if i'm interpreting correctly